service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -110,33 +112,35 @@ func newTransferQueueActiveTaskExecutor(
 func (t *transferQueueActiveTaskExecutor) Execute(
 	ctx context.Context,
 	executable queues.Executable,
-) (metrics.Scope, error) {
+) (metrics.MetricProvider, error) {
 
 	task := executable.GetTask()
-	scope := t.metricsClient.Scope(
-		tasks.GetActiveTransferTaskMetricsScope(task),
-		getNamespaceTagByID(t.registry, task.GetNamespaceID()),
+	taskType := queues.GetActiveTransferTaskTypeTagValue(task)
```

Comment:
Good point. It should be doable, but I will need to do some string manipulation so that `"TransferActive" + task.TaskType()` is equal to the existing tag value.

Reply:
I changed the implementation of those method to be string manipulation. But there're a few special handling in the methods to keep the value same as today. I personally think it's ok to make a breaking change here and use a different tag value. Or I can change the proto definition of TaskType. Let me know your thoughts. 

The string manipulation will be much slower than the hardcoded lookup though.
```
BenchmarkTaskTypeTagValue-10    	88972023	        13.50 ns/op	       0 B/op	       0 allocs/op
```
```
BenchmarkTaskTypeTagValue-10    	10984756	       105.2 ns/op	     168 B/op	       2 allocs/op
```

Reply:
Synced offline. Will revert to the original table look up for now and create an issue to follow up. 

Landing this PR first to unblock the remaining work stream.